src/Activity.js

```diff
@@ -1,60 +1,62 @@
-class Activity {
-  constructor(activityData) {
-    this.activityData = activityData
+import Fitlit from "./Fitlit";
+
+class Activity extends Fitlit {
```

Comment:
I saw where you mentioned looking for a better name for this in your PR. While we'll want to rename this later, I think for the purposes of having a working product, it's fine this way for now.